src/server/game/Server/WorldSocketMgr.cpp

```diff
@@ -51,6 +51,8 @@ WorldSocketMgr::~WorldSocketMgr()
     delete _instanceAcceptor;
 }
 
+int const boost::asio::socket_base::max_connections;
```

Comment:
Its required to fix clang build, was wondering too.
http://stackoverflow.com/questions/4891067/weird-undefined-symbols-of-static-constants-inside-a-struct-class
http://stackoverflow.com/questions/9213985/linking-errors-with-static-const-values-referring-to-enum-in-alternative-namespa